data_steward/cdr_cleaner/cleaning_rules/remove_participant_data_past_deactivation_date.py

```diff
@@ -0,0 +1,164 @@
+"""
+Ensures there is no data past the deactivation date for deactivated participants.
+
+Original Issue: DC-686
+
+The intent is to sandbox and drop records dated after the date of deactivation for participants
+who have deactivated from the Program.
+"""
+
+# Python imports
+import logging
+
+# Project imports
+import common
+from utils import bq
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+import utils.participant_summary_requests as psr
+import retraction.retract_deactivated_pids as rdp
+from constants.retraction.retract_deactivated_pids import DEACTIVATED_PARTICIPANTS
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+# Third-Party imports
+import google.cloud.bigquery as gbq
+
+LOGGER = logging.getLogger(__name__)
+
+DEACTIVATED_PARTICIPANTS_COLUMNS = [
+    'participantId', 'suspensionStatus', 'suspensionTime'
+]
+
+
+def remove_ehr_data_queries(client, api_project_id, project_id, dataset_id,
+                            sandbox_dataset_id):
+    """
+    Sandboxes and drops all EHR data found for deactivated participants after their deactivation date
+
+    :param client: BQ client
+    :param api_project_id: Project containing the RDR Participant Summary API
+    :param project_id: Identifies the project containing the target dataset
+    :param dataset_id: Identifies the dataset to retract deactivated participants from
+    :param sandbox_dataset_id: Identifies the sandbox dataset to store records for dataset_id
+    :returns queries: List of query dictionaries
+    """
+    # gets the deactivated participant dataset to ensure it's up-to-date
+    df = psr.get_deactivated_participants(api_project_id,
+                                          DEACTIVATED_PARTICIPANTS_COLUMNS)
+
+    # To store dataframe in a BQ dataset table named _deactivated_participants
+    destination_table = f'{sandbox_dataset_id}.{DEACTIVATED_PARTICIPANTS}'
+    psr.store_participant_data(df, project_id, destination_table)
+
```

Comment:
Will this be reused for fitbit?

Reply:
we decided to run the API request twice for now.